src/components/TimeoutNotification/TimeoutNotification.js

```diff
@@ -13,13 +13,19 @@ class TimeoutNotification extends Component {
 
   componentDidMount() {
     document.body.addEventListener('touchstart', this.dismissMessage);
+    document
+      .getElementById('notificationBar')
```

Comment:
Instead of adding event listeners to dom elements (potentially not playing nice with react) could we use [stopPropagation](https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation) to prevent the event from reaching the item?

It'd look something like

```
notificationTouch(event) {
  event.stopPropagation();
  this.dismissMessage();
}
<div
  className="'timeout-notification--notification--..."
  onTouchStart={this.notificationTouch}
/>
```

It can be a cannon, but could be what we want in this scenario